src/main/java/rs/teslaris/core/service/impl/document/DocumentPublicationServiceImpl.java

```diff
@@ -197,7 +200,7 @@ private void indexDocumentFilesContent(Document document, DocumentPublicationInd
         index.setFullTextOther("");
         document.getFileItems().forEach(documentFile -> {
             var file = documentFileService.findDocumentFileIndexByDatabaseId(documentFile.getId());
-            index.setFullTextSr(index.getFullTextSr() + "" + file.getPdfTextSr());
+            index.setFullTextSr(index.getFullTextSr() + file.getPdfTextSr());
```

Comment:
do we need here + " " + in between? (to permit concatenation of the last word in getFullTextSr() and the first word in getPdfTextSr()? Or there is a space at the end of returned string in the first method?

Reply:
I wanted to type in " " but made a typo. Great catch!